db/migrate_cards/20160514205559_add_cso_metrics.rb

```diff
@@ -0,0 +1,81 @@
+# -*- encoding : utf-8 -*-
+
+class AddCsoMetrics < Card::Migration
+  # I'm using the row number from the spreadsheet for an easier translation
+  # number => {{metric}}
+  # $number => {{metric | year: 2005}}
+  # #number => {{metric | year: 2006..0}}
+  DATA =
+    {
+      70 => ['Maximum Allowable OECD CO2 Emissions', nil],
+      71 => ['Maximum Allowable OECD CO2 Emissions - Ratio to Baseline', nil],
+      72 => ['GDP of OECD', nil],
+
+      75 => ['Value-added Contributions to GDP (C2GDP)', nil],
+      76 => ['Annual CO2 Emissions', nil],
+      77 => ['Gross Revenue', nil],
+
+      80 => ['Cumulative CO2 Emissions Context-Free Score', '86/89'],
+      81 => ['Annual CO2 Emissions Context-Based Relative Score', '(76/75)/95'],
+      82 => ['Annual CO2 Emissions Context-Based Absolute Score', '76/91'],
+      83 => ['Cumulative CO2 Emissions Context-Based Absolute Score', '86/92'],
+
+      85 => ['CO2 Emissions Relative to Gross Revenue', '76/77'],
+      86 => ['Cumulative CO2 Emissions', '#85'],
+
+      88 => ['Maximum Allowable Annual CO2 Emissions', '$76*71'],
+      89 => ['Maximum Allowable Cumulative CO2 Emissions', '#88'],
+
+      91 => ['Annual CO2 Emission Targets', '95*75'],
+      92 => ['Cumulative CO2 Emission Targets', '#91'],
+
+      94 => ['Maximum Allowable Annual CO2 Emissions per C2GDP', '$76/$75*71'],
```

Comment:
this is what I have in the spreadsheet, but rearranging it to ($76*71)/$75 might make it easier to get your head around.